.github/workflows/develop.yml

```
@@ -152,7 +152,7 @@ jobs:
       - depend
       - GNU
       - Intel
-      - Nvidia
+#      - Nvidia
```

We should still wait for it to complete, we should just not use it to determine the return value of the final check
```suggestion
      - Nvidia
```